lib/validators.js

```diff
@@ -104,9 +104,12 @@ const Validators = ({
      * RFC 2616 Section 14.9.
      */
     onlyPrivate: (response, evaluator) => {
-      if ((response.headers['cache-control'] || '').match(/private/) && !evaluator.storePrivate) {
-        evaluator.flagStorable(false);
-        return true;
+      if ((response.headers['cache-control'] || '').match(/private/)) {
```

Comment:
same here.
iirc, response evaluator is assumed storable by default – and evaluators only have a say when they detect conditions that make the response not storable

Reply:
If we want to store private data, we provide a flag "storePrivate". The only purpose of this change is to notify the caller that this is actually a private data and it should be treated as such. The logic hasn't been changed.